### PR TITLE
PSS/E. Discard controls regulating isolated buses. Do not remove comments inside quotes.

### PR DESCRIPTION
**Pull request by marqueslanauja**
Signed-off-by: José Antonio Marqués <marquesja@aia.es>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
An exception is thrown when a regulating terminal is not found.
Comments inside quotes were not expected.

**What is the new behavior (if this is a feature change)?**
Controls regulating isolated buses are discarded as there is not regulating terminal associated to. A warning is logged.
Only comments outside quotes are removed.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: psse/psse-converter/src/main/java/com/powsybl/psse/converter/GeneratorConverter.java [hunk 1]
```diff
@@ -100,8 +103,8 @@ private static Terminal defineRegulatingTerminal(PsseGenerator psseGenerator, Ne
             }
         }
         if (regulatingTerminal == null) {
-            throw new PsseException("PSSE. Generator " + defaultRegulatingBusId + "-"
-                + getGeneratorId(defaultRegulatingBusId, psseGenerator) + ". Unexpected regulatingTerminal.");
+            String generatorId = getGeneratorId(defaultRegulatingBusId, psseGenerator);
+            LOGGER.warn("PSSE. Generator {}. Regulating terminal is not assigned as the bus is isolated", generatorId);
```
Comment (annetill) [inline psse/psse-converter/src/main/java/com/powsybl/psse/converter/GeneratorConverter.java:107]: I am not sure that you need to mention `PSSE`, the LOGGER makes it for you.

File: psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java [hunk 2]
```diff
@@ -117,8 +120,8 @@ private static Terminal defineRegulatingTerminal(PsseSwitchedShunt psseSwitchedS
             }
         }
         if (regulatingTerminal == null) {
-            throw new PsseException("PSSE. SwitchedShunt " + defaultRegulatingBusId + "-"
-                + defineShuntId(psseSwitchedShunt, version) + ". Unexpected regulatingTerminal.");
+            String shuntId = defineShuntId(psseSwitchedShunt, version);
+            LOGGER.warn("PSSE. SwitchedShunt {}. Regulating terminal is not assigned as the bus is isolated", shuntId);
```
Comment (annetill) [inline psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java:124]: Same remark, you can remove the `PSSE` label.

**Review by annetill — APPROVED** [review]
Just small changes requested for LOGGER but it depends also if you have made that in the rest of the code or not. If yes, you can keep it and I will approve the PR as it is.

File: psse/psse-converter/src/main/java/com/powsybl/psse/converter/GeneratorConverter.java (hunk 1 above)
Comment (marqueslanauja) [reply psse/psse-converter/src/main/java/com/powsybl/psse/converter/GeneratorConverter.java]: Done

File: psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java (hunk 2 above)
Comment (marqueslanauja) [reply psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java]: Done

**Pull request merged**